scripts/scil_flip_or_swap_gradients_axis.py

```diff
@@ -0,0 +1,104 @@
+#! /usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Flip (ex, x --> -x) or swap (ex, x <-> y) chosen axes of the gradient sampling
+matrix. Result will be saved in the same format as input gradient sampling
+file.
+"""
+import argparse
+import os
+
+import numpy as np
+
+from scilpy.io.utils import (add_overwrite_arg, assert_inputs_exist,
+                             assert_outputs_exist)
+from scilpy.gradients.bvec_bval_tools import (flip_gradient_sampling,
+                                              swap_gradient_axis)
+from scilpy.utils.util import str_to_index
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter,
+                                description=__doc__)
+
+    p.add_argument('in_gradient_sampling_file',
+                   help='Path to gradient sampling file. (.bvec or .b)')
+
+    p.add_argument('out_gradient_sampling_file',
+                   help='Where to save the flipped gradient sampling file.')
+
+    # Note: We can't ask for 3 separate nargs because -x is understood as
+    # another option -x.
+    # Note that here if user sends "-xyz" (forgetting the spaces) the error
+    # will be: parser.error:  argument --final_order: expected one argument
+    p.add_argument('--final_order', metavar='"axis axis axis"',
+                   help="The final order of the axis, compared to original "
+                        "order. \n"
+                        "Choices: ['x', 'y', 'z', '-x', '-y', '-z']\n"
+                        "Ex: to only flip y: --final_order x -y z.\n"
+                        "Ex: to only swap x and y: --final_order y x z.\n"
+                        "Ex: to first flip x, then permute all three axes: "
+                        "--final_order z -x y.")
+
+    gradients_type = p.add_mutually_exclusive_group(required=True)
```

Comment:
I don't understand if you put .b it's mrtrix if it's bvec it's FSL no need for this argument.

Reply:
Agreed. You already look for it below.

Reply:
I agree. It was already there... Changed.